cg/cli/base.py

```diff
@@ -65,22 +65,20 @@ def base(
 @base.command()
 @click.option("--reset", is_flag=True, help="reset database before setting up tables")
 @click.option("--force", is_flag=True, help="bypass manual confirmations")
-@click.pass_obj
-def init(context: CGConfig, reset: bool, force: bool):
+def init(reset: bool, force: bool):
```

Comment:
```suggestion
def init_statusdb(reset: bool, force: bool):
```
ignore if out of scope

Reply:
Leaving it for now